src/client/entity-editor/common/entity.js

```diff
@@ -47,6 +48,7 @@ function Entity(
 				disambiguation &&
 				<span className="disambig"><small>({disambiguation})</small></span>
 			}
+			<span style={{float: 'right'}}>{language}</span>
```

Comment:
Maybe worth adding a `className="text-muted small"` there, just to make it a bit more distinct.

Reply:
Okay.